python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -452,7 +452,8 @@ def getMjd(self, butler, dataId, timescale=dafBase.DateTime.UTC):
 
     def getFilter(self, butler, dataId):
         """Determine the filter from a data identifier"""
-        return dataId[self.config.filter]
+        filt = butler.queryMetadata('raw', ['filter'], dataId)[0]
```

Comment:
This seems like a regression --- hard-coding `filter` instead of using what we're told to in the `config`.